main.c

```diff
@@ -86,15 +68,16 @@ int main(int argc, char** argv) {
     I2C1_Initialize();
     
     LED_init();
-    LinAc_init();
+    LINAC_init();
     
-    // Set DACs
-    setDACs(LOW_VAL, HIGH_VAL);
+    set_DACs();
     
-    closeVent();
+    //close_vent();
     while (1){
-        BatV = GET_BATTERY_VOLTAGE;
-        if(RB4){
+        BatV = ADCC_GetSingleConversion(channel_VBAT);
```

Comment:
In the future, add a comment or name the variables so that the units of the output are clear. These are raw adc values that just happen to correspond to mV due to the configuration of this particular ADC.